Makefile

```diff
@@ -683,6 +705,11 @@ FILES_TO_CLEAN+=install/consul/istio.yaml \
                 install/kubernetes/istio.yaml \
                 samples/bookinfo/platform/consul/bookinfo.sidecars.yaml \
 
+generate_yaml_cni: export ENABLE_ISTIO_CNI=true
```

Comment:
make the hub customizable - and I would expect the CI/CD should first build it ( checkout and build the repo, or have it as a dep ), or a 'daily' build of the CNI is used..

Reply:
@costin - i will look into making this customizable, but here is the strategy we had for this testing.  
For istio/istio change spawn at least 1 e2e test using the CNI based on a known good CNI label.  In other words we did not want istio/istio based PRs to be rebuilding the CNI based on latest.   Additionally we are adding gates to the CNI repo that takes a known good istio/istio label and runs e2e tests with the CNI built from the PR.   IOW 
istio/istio gate with known good CNI label
istio/cni gate with known good istio label
as we migrate to hte CNI being the default than probably we should run both gates with the latest daily.  If you have a different opinion let us know.